plugins/utils/which-key.nix

```diff
@@ -55,257 +277,317 @@ with lib;
+      marks = defaultNullOpts.mkBool true ''
+        Shows a list of your marks on `'` and `` ` ``.
+      '';
 
-      registers = helpers.defaultNullOpts.mkBool true ''shows your registers on " in NORMAL or <C-r> in INSERT mode'';
+      registers = defaultNullOpts.mkBool true ''
+        Shows your registers on `"` in NORMAL or `<C-r>` in INSERT mode.
+      '';
 
       spelling = {
-        enabled = helpers.defaultNullOpts.mkBool true "enabling this will show WhichKey when pressing z= to select spelling suggestions";
-        suggestions = helpers.defaultNullOpts.mkInt 20 "how many suggestions should be shown in the list?";
+        enabled = defaultNullOpts.mkBool true ''
+          Enabling this will show WhichKey when pressing `z=` to select spelling suggestions.
+        '';
+        suggestions = defaultNullOpts.mkInt 20 ''
+          How many suggestions should be shown in the list?
+        '';
       };
 
       presets = {
-        operators = helpers.defaultNullOpts.mkBool true "adds help for operators like d, y, ...";
-        motions = helpers.defaultNullOpts.mkBool true "adds help for motions";
-        textObjects = helpers.defaultNullOpts.mkBool true "help for text objects triggered after entering an operator";
-        windows = helpers.defaultNullOpts.mkBool true "default bindings on <c-w>";
-        nav = helpers.defaultNullOpts.mkBool true "misc bindings to work with windows";
-        z = helpers.defaultNullOpts.mkBool true "bindings for folds, spelling and others prefixed with z";
-        g = helpers.defaultNullOpts.mkBool true "bindings for prefixed with g";
+        operators = defaultNullOpts.mkBool true "Show WhichKey for operators like `d`, `y`, ...";
+        motions = defaultNullOpts.mkBool true "Show WhichKey for motions.";
+        text_objects = defaultNullOpts.mkBool true "Show WhichKey for text objects triggered after entering an operator.";
+        windows = defaultNullOpts.mkBool true "Show WhichKey for default bindings on `<c-w>`.";
+        nav = defaultNullOpts.mkBool true "Misc bindings to work with windows.";
+        z = defaultNullOpts.mkBool true "Show WhichKey for folds, spelling and other bindings prefixed with `z`.";
+        g = defaultNullOpts.mkBool true "Show WhichKey for bindings prefixed with `g`.";
       };
     };
 
-    operators = helpers.defaultNullOpts.mkAttrsOf types.str { gc = "Comments"; } ''
-      add operators that will trigger motion and text object completion
-      to enable all native operators, set the preset / operators plugin above
-    '';
+    win = {
+      no_overlap = defaultNullOpts.mkBool true "Don't allow the popup to overlap with the cursor.";
 
-    keyLabels = helpers.defaultNullOpts.mkAttrsOf types.str { } ''
-      override the label used to display some keys. It doesn't effect WK in any other way.
-    '';
+      border = defaultNullOpts.mkBorder "none" "which-key" "";
```

Comment:
Any idea what the description should be here? Also, have we double checked `mkBorder` is still the correct type for the new v3 format?

Reply:
Sorry, forgot this one. It's basically just the border property inherited from neovim's Win type iirc